src/test/java/redis/clients/jedis/commands/jedis/ClientCommandsTest.java

```diff
@@ -38,6 +42,11 @@ public class ClientCommandsTest extends JedisCommandsTestBase {
 
   private Jedis client;
 
+  @Rule
+  public RedisVersionRule versionRule = new RedisVersionRule(endpoint);
+  @Rule
+  public EnabledOnCommandRule enabledOnCommandRule = new EnabledOnCommandRule(endpoint);
+
```

Comment:
Do we need these here anymore? As these conditions are already in the super-class, JedisCommandsTestBase.